### PR TITLE
A small change that enables direct lazygit directory config

### PR DESCRIPTION
**Pull request by Liberatys**
This small change allows the usage of .config/lazygit instead of .config/jesseduffield/lazygit. \
Not because we don't like to work that jesse is doing but because terminal users are way to lazy to write another directory name if it can be omited ^^.

This change uses the fact that the package for concatenating the directory name is omit empty vendor names from the path

### CODE REVIEW

**Comment by Liberatys** [comment]
> From: Jesse
> 
> Seems to me like we should be first seeing if the legacy dir exists, and if so, use that, otherwise use the non-namespaces one. Does that sound right?

This is up to you ^^ The order doesn't really matter. But I see your point of using the new "default" if the legacy director does not exist

File: main.go [hunk 1]
```diff
@@ -100,7 +100,7 @@ func main() {
 	}
 
 	if configDirFlag {
-		fmt.Printf("%s\n", config.ConfigDir())
+		fmt.Printf("%s\n", config.ConfigDir("jesseduffield"))
```
Comment (jesseduffield) [inline main.go:103]: seems to me that this should use `SelectDefaultConfiguration`, does that sound right?

File: pkg/config/app_config.go [hunk 2]
```diff
@@ -81,21 +81,28 @@ func NewAppConfig(name, version, commit, date string, buildSource string, debugg
 	return appConfig, nil
 }
 
-func ConfigDir() string {
+func SelectDefaultConfiguration() string {
```
Comment (jesseduffield) [inline pkg/config/app_config.go:84]: What are your thoughts on renaming this to `ConfigDir` and then having the existing ConfigDir become a private method called `configDirForVendor`

File: pkg/config/app_config.go [hunk 3]
```diff
@@ -81,21 +81,28 @@ func NewAppConfig(name, version, commit, date string, buildSource string, debugg
...
+	return configDirectory
+}
+
+func ConfigDir(vendor string) string {
 	envConfigDir := os.Getenv("CONFIG_DIR")
 	if envConfigDir != "" {
 		return envConfigDir
 	}
-
 	// chucking my name there is not for vanity purposes, the xdg spec (and that
```
Comment (jesseduffield) [inline pkg/config/app_config.go:98]: we can remove this comment now

**Review by jesseduffield — CHANGES_REQUESTED** [review]
Thanks for making this :) I've left a couple comments

File: pkg/config/app_config.go (hunk 2 above)
Comment (Liberatys) [reply pkg/config/app_config.go]: 👍 then the change in main.go can be reverted to the old state

**Comment by Liberatys** [comment]
Updated as per your comments.
Also opened https://github.com/jesseduffield/lazydocker/pull/245 against lazydocker for the same change

**Review by jesseduffield — APPROVED** [review]
Functionally tested and it works. Great job!

**Pull request merged**